### PR TITLE
Update Mongo Go driver tests configuration

### PR DESCRIPTION
**Pull request by noisersup**
Refs #134.

### CODE REVIEW

File: tests/mongo-go-driver.yml [hunk 1]
```diff
@@ -67,14 +71,35 @@ results:
...
+      # skip if CRYPT_SHARED_LIB_PATH is not set
+      - go.mongodb.org/mongo-driver/mongo/TestClient/mongocryptd_or_crypt_shared
+
+      # skip if no compressor specified
+      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestCompression
+
+      # skip because authentication is required
+      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestSCRAM
+
+      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestInsert
```
Comment (noisersup) [inline tests/mongo-go-driver.yml:93]: I'm not sure how to comment that line.
The test just [skips on the first line of it](https://github.com/mongodb/mongo-go-driver/blob/master/x/mongo/driver/integration/insert_test.go#L20)

Comment (AlekSi) [reply tests/mongo-go-driver.yml:93]: I think that could be something like "Always skipped"

**Review by AlekSi — COMMENTED** [review]
Looks good. Please add those comments about skipped tests

**Comment by AlekSi** [comment]
Any ideas how to improve that part shared between configurations?

**Comment by noisersup** [comment]
In that topic I was thinking about a few approaches:
- Move the `skip` section to seperate block and adjust a code to parse it in both databases. In fact I suppose that this [common field](https://github.com/FerretDB/dance/blob/main/internal/config.go#L39) could be an attempt to do that (correct me if I'm wrong)
- A yaml generation (something like jsonnet where we can just declare a variable and "compile" to yaml). I can make a little research on that if needed.

**Comment by noisersup** [comment]
Also if we are talking about improvements. Is there anything that stops us from automating submodule updates?

**Comment by AlekSi** [comment]
Dependabot already does that: https://github.com/FerretDB/dance/blob/edf7f8560a91eeadabf39b25b189c13aa392ce88/.github/dependabot.yml#L25-L31

**Comment by AlekSi** [comment]
See #139, for example

**Comment by AlekSi** [comment]
See #142

**Review by AlekSi — APPROVED** [review]

**Pull request merged**